tests/functional/c/const_class_attribute_same_name.py

```diff
@@ -0,0 +1,14 @@
+"""Test module-level constants with class attribute same name
+Regression test for #10719.
+"""
+# pylint: disable=missing-docstring, too-few-public-methods, redefined-builtin
```

Comment:
Most of the tests under `/functional` are organized by the message they should or shouldn't emit. Please move these test cases to the existing `invalid_name_module_level.py`